packages/select/src/ResourceSelect.js

```diff
@@ -81,6 +81,18 @@ const ResourceSelect = ({
         },
```

Comment:
We actually should remove the original `...rest.parameters` spread above so we can check first if we need to spread or call the function.